.github/workflows/publish.yaml

```diff
@@ -24,21 +29,23 @@ jobs:
       - name: Checkout code-server
         uses: actions/checkout@v3
 
-      - name: Get version
-        id: version
-        run: echo "::set-output name=version::$(jq -r .version package.json)"
-
       - name: Download npm package from release artifacts
         uses: robinraju/release-downloader@v1.5
         with:
           repository: "coder/code-server"
-          tag: v${{ steps.version.outputs.version }}
+          tag: ${{ github.event.inputs.version || github.ref_name }}
           fileName: "package.tar.gz"
           out-file-path: "release-npm-package"
 
+      - name: Get and set VERSION
```

Comment:
For a second I forgot why we needed to do this.  :laughing:   We might want to leave a comment that this exists just so we can strip out the `v` (same elsewhere this is used).

Reply:
Gooooood idea!